gamemodes/prop_hunt/gamemode/sh_lps.lua

```diff
@@ -0,0 +1,55 @@
+CreateConVar("lps_enable", "1", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Give the last prop alive a weapon?")
+CreateConVar("lps_weapon", "random", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Give which weapon (\"random\" from the config or by class name)?")
+CreateConVar("lps_halo_enable", "1", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw a halo effect around the last prop?")
+CreateConVar("lps_halo_color", "#14FA00", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw a halo effect of what color (\"rainbow\" or by hex code)?")
+CreateConVar("lps_halo_walls", "0", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw the halo effect through walls?")
+CreateConVar("lps_trail_enable", "1", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw a trail behind the last prop?")
+CreateConVar("lps_trail_color", "#FFFFFF", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw a trail of what color (by hex code)?")
+CreateConVar("lps_trail_texture", "trails/laser", {FCVAR_SERVER_CAN_EXECUTE, FCVAR_REPLICATED, FCVAR_NOTIFY}, "Draw a trail with what texture?")
+
+math.randomseed(os.time())
+
+PHE.LPS = {}
+
+function lpsWeaponSetup()
+	if !(GetConVar("lps_weapon"):GetString() == "random") then
+        PHE.LPS.WEAPON = GetConVar("lps_weapon"):GetString()
+    end
+end
+lpsWeaponSetup()
+cvars.AddChangeCallback("lps_weapon", lpsWeaponSetup())
+
+AddCSLuaFile("sh_lps_config.lua")
```

Comment:
You can remove the `sh_lps_config.lua` and it'll imply the current file.

Reply:
I used to think that it worked that way, but PH:E's code has to specifically `AddCSLuaFile` and `include` everything except `init.sh` and `cl_init.sh`. Ultimately, this will become irrelevant, as the LPS files will be part of an entirely separate addon, which will work more like what you said.

Reply:
Ah my bad, it's just for `autorun`